### DISCUSSION TITLE
Choria builds for Debian Bookworm

### DISCUSSION THREAD

**Discussion by yakatz** [choria-io/general#39] [General]
We are using the Puppet module to install Choria. We recently upgraded some systems to Debian Bookworm and Puppet now creates the file `/etc/apt/sources.list.d/choria-release.list` with the contents:
```
# This file is managed by Puppet. DO NOT EDIT.
# Choria Orchestrator Releases
deb mirror://mirrorlists.choria.io/apt/release/debian/bookworm/mirrors.txt debian bookworm
```

It appears there isn't a release available for bookworm yet and I don't see any discussion or procedure related to having one. Any way of getting a release for that?

**Comment by smortex** [comment]
Hey @yakatz,

I guess the topic came on Slack as I can't find it here on GitHub.  Basically, we are waiting for Debian 12 packages of puppet-agent so that we can work on "proper" packaging. In the meantime, it might be possible to run packages you get from an older repo.

Debian 12 packaging of puppet-agent has not started yet according to:
https://tickets.puppetlabs.com/browse/PA-5549

**Comment by timdeluxe** [comment]
Hi @smortex 
the puppet agent packages for Debian 12 just recently were released: https://www.puppet.com/docs/puppet/7/release_notes_puppet#release_notes_puppet_x-7-29-0
Can the choria builds for bookworm picked up now please?

**Comment by ripienaar** [comment]
I've initiated a build for the packager container now.  But for a working choria deb we have to wait for next release, hopefully not too long

**Comment by ripienaar** [comment]
Merged release tooling here https://github.com/choria-io/go-choria/issues/2127
